spring-cloud-gcp-samples/spring-cloud-gcp-vision-api-sample/src/main/resources/static/index.html

```diff
@@ -31,5 +31,17 @@ <h1>Text Extraction</h1>
     </form>
 </div>
 
+<div>
+    <h1>Text Extraction PDF</h1>
+    <p>Read and extract the text from a PDF:</p>
+    <form action="/extractTextFromPdf">
+        Web URL of a PDF to analyze:
```

Comment:
This might be a good place to also mention this is meant for small PDF files less than 5 pages.

Reply:
I added a small comment above regarding the maximum pages.